components/panel/body.js

```diff
@@ -52,7 +52,14 @@ class PanelBody extends Component {
 							aria-expanded={ isOpened }
 						>
 							<Dashicon icon={ icon } />
-							{ title }
+							<span className="components-panel__body-title__text-span">
+								{ title }
+							</span>
+							{ colorValue &&
```

Comment:
Is `colorValue` a concern of a general-purpose panel component? Maybe this ought to be a specialized component which extends the base panel component behaviors?

Reply:
Yes, I agree color is not a concern for a generic panel. I was not seeing the best way to extend and it looked like I needed to pass a prop. I end up making the panel body component receive an optional area to render at the side of the title. I created a PanelColor component that uses the panel body, rendering the correct area for the color.